TeacherWorkout.Api/TeacherWorkout.Api.csproj

```diff
@@ -2,21 +2,24 @@
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
+    <UserSecretsId>af66c3e1-ddfc-4130-9207-6a6bdc33931e</UserSecretsId>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="GraphQL.Server.Transports.AspNetCore.SystemTextJson" Version="5.0.2" />
-    <PackageReference Include="GraphQL.Server.Ui.Graphiql" Version="5.0.2" />
+    <PackageReference Include="GraphQL.Server.Ui.Playground" Version="5.2.0" />
```

Comment:
Please also update the readme to point to the graphql docs

http://localhost:5000/ui/playground